### DISCUSSION TITLE
Correct volume transition between "inactive" and "active" window state

### DISCUSSION THREAD

**Discussion by Kyorie** [ppy/osu#21852] [Ideas] [answered]
It's a smooth volume change when going unactive, but when going back to "active" window state - the volume increase is instant which is scary. Also I'd make active-unactive transition a bit faster (500ms less?) and create a toggle for using "Master (window inactive)" to always have the same volume.
![image](https://user-images.githubusercontent.com/47324490/209727962-c0cf4bd9-e804-49e7-a6ac-ff392265e6e6.png)

**Comment by peppy** [comment, marked as answer]
This was an intentional choice. It's also not instant (400ms for resume, 4000ms for suspend).

> create a toggle for using "Master (window inactive)" to always have the same volume.

Set it to 100% for this. It's a relative adjustment.
